private/buf/cmd/buf/buf_test.go

```diff
@@ -482,6 +482,20 @@ func TestFailCheckBreaking5(t *testing.T) {
 	)
 }
 
+func TestFailCheckBreaking6(t *testing.T) {
```

Comment:
The test for this should be in https://github.com/bufbuild/buf/blob/main/private/bufpkg/bufcheck/bufbreaking/bufbreaking_test.go